### PR TITLE
Alphabetised dispenser .yaml files

### LINKED ISSUE #3000: Reagent dispenser entries should be alphabetised

### PR DESCRIPTION
**Pull request by Peptide90**
closes #3000

I messed up on #3253 so closed that in favour of this.

### CODE REVIEW

**Review by metalgearsloth — APPROVED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/snack.yml [hunk 1]
```diff
@@ -4,9 +4,8 @@
   animationDuration: 0.5
   spriteName: snack
   startingInventory:
-    FoodSpaceTwinkie: 8
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/VendingMachines/snack.yml:7]: Was this meant to be removed?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/snack.yml (hunk 1 above)
Comment (Peptide90) [reply Resources/Prototypes/Catalog/VendingMachines/snack.yml:7]: Must have missed it off the sort. Fixed thanks.

File: Resources/Prototypes/Catalog/VendingMachines/cola.yml [hunk 2]
```diff
@@ -6,11 +6,10 @@
...
   startingInventory:
     DrinkColaCan: 10
+    DrinkGrapeCan: 10
     DrinkIceTeaCan: 10
     DrinkLemonLimeCan: 10
     DrinkGrapeCan: 10
     DrinkSpaceMountainWindCan: 10
     DrinkSpaceUpCan: 10
     DrinkStarkistCan: 10
-    DrinkThirteenLokoCan: 10
```
Comment (PaulRitter) [inline Resources/Prototypes/Catalog/VendingMachines/cola.yml:15]: grafik

**Review by PaulRitter — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/VendingMachines/cola.yml (hunk 2 above)
Comment (Peptide90) [reply Resources/Prototypes/Catalog/VendingMachines/cola.yml]: I am ashamed. No idea how that happened from copying to excel, hitting sort and copying back.

Resolved anyhow.

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**